source/_integrations/plugwise.markdown

```diff
@@ -33,17 +33,21 @@ The password can be found on the bottom of your Smile, it should consist of 6 ch
+This integration will show all Plugwise entities present in your Plugwise configuration. In addition you will see a 'Smile' entity representing your central Plugwise gateway (i.e. the Smile, Smile P1 or Adam).
 
-For example, if you have an Adam setup with a Lisa named 'Living' and a Tom named 'Bathroom' these will show up as individual devices. For setups where an auxiliary heater (or heater/cooler) is present there will be an additional device representing it called 'Auxiliary'.
+For example, if you have an Adam setup with a Lisa named 'Living' and a Tom named 'Bathroom' these will show up as individual entities. When there are more (than one) Plugwise thermostats present, an "Auxiliary Device" will be added representing the active heating/cooling device present in your climate system. If you have Plugs (as in, pluggable switches connecting to an Adam) those will be discovered as `switch`es. Various other measures of your setup will be available as `sensor`s or `binary_sensor`s.
 
-Centralized measurements such as 'power' for a P1, 'outdoor_temperature' on Anna or Smile will be assigned to your gateway device. Auxiliary Heater(/Cooler) measurements such as 'boiler_temperature' will be assigned to the Auxiliary entity.
+Centralized measurements such as 'power' for a P1, 'outdoor_temperature' on Anna or Smile will be assigned to your gateway entity. Auxiliary Heater(/Cooler) measurements such as 'boiler_temperature' will be assigned to the Auxiliary entity.
 
 ## Configuration
 
-To set up this integration, click Configuration in the sidebar and then click Integrations. Add a new integration using the "+" button in the lower right corner and look for 'Plugwise'. Click configure and you will be presented with a dialog requesting the Smile ID or password of your Smile and it's IP address. After you click submit, you will have the opportunity to select the area(s) where individual Smile appliances are located.
+The Plugwise Smile(s) present in your network will be automatically detected via Zeroconf discovery and will be shown on the Integrations-page. To set up an integration, click CONFIGURATION on the discovered Entity and you will be presented with a dialog requesting the Smile password. After you click submit, you will have the opportunity to select the area(s) where individual Smile appliances are located.
 
-Depending on your `climate` setup an auxiliary entitiy will be added when there is information available about such Plugwise devices. If you have "plug"s (as in, pluggable switches that come with an Adam) those will be discovered as `switch`es. Various other measures of your setup will be available as `sensor`s or `binary_sensor`s.
+PLEASE NOTE 1: the Unique ID(s) of the Plugwise integration(s) has(have) changed due to the introduction of automatic discovery. If you have already configured the Plugwise integration(s) via the "Integrations --> Plus-button"-method, you can ignore the newly discovered Plugwise integration(s).
```

Comment:
This is a breaking change that should be part of the release notes, not the documentation.

Reply:
This can be removed, it isn't changing of unique ID. The way zeroconf detects and registers is different. There is nothing breaking here :)

Reply:
Ok, noted, will be removed.

Reply:
It is true you can ignore it ... for clarification sake ... zeroconf provides the smile id (i.e. `smile01234`) but without the authentication password we can't fish it out of the XML. On top of that, not all devices have it in their XMLs.

Reply:
Removed.